src/Uno.UWP/ApplicationModel/Activation/LaunchActivatedEventArgs.cs

```diff
@@ -18,7 +18,6 @@ internal LaunchActivatedEventArgs(ActivationKind kind, string arguments)
 			Kind = kind;
 		}
 
-		[NotImplemented]
```

Comment:
Still `[NotImplemented]` for iOS & Wasm...

Reply:
,... at least Wasm

Reply:
Fixed